ide/web/lib/files_mock.dart

```diff
@@ -104,6 +105,9 @@ class MockFileSystem implements FileSystem {
     entry._modificationTime = new DateTime.fromMillisecondsSinceEpoch(
         entry._modificationTime.millisecondsSinceEpoch + 1);
   }
+
+  // Added to satisfy the analyzer.
+  JsObject blink_jsObject;
```

Comment:
Yeah, that would follow the style guide better. In this case, the parent object we're using as an interface has a newly introduced `blink_jsObject` field, so we need to as well -